### PR TITLE
feat: the description box should expand with text

### PR DESCRIPTION
**Pull request by iMac7**
Relates to:  #7819

### Description:
Replaced the textinput component with a textarea. I am not sure about the actual maximum rows expected, so I just picked what looked right, feel free to change that.

image

**Note:** The 'add chart' modal has not been done yet, because of possible merge conflicts with #7807 , which involves modifying the same text input component to work like the textarea I have implemented above.

### Reviewer actions

- [x] I have manually tested the changes in the preview environment
- [x] I have reviewed the code
- [x] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by joaoviana** [comment]
hi @iMac7 , thanks for your contribution! I've updated your PR description to not automatically close the issue since: 
* it should be changed on a few more places: the charts and dashboards create and update modals - basically anywhere where you can update the description

File: packages/frontend/src/components/common/modal/DashboardCreateModal.tsx [hunk 1]
```diff
@@ -147,10 +148,13 @@ const DashboardCreateModal: FC<DashboardCreateModalProps> = ({
                             required
                             {...form.getInputProps('dashboardName')}
                         />
-                        <TextInput
+                        <Textarea
                             label="Dashboard description"
                             placeholder="A few words to give your team some context"
                             disabled={isCreatingDashboard}
+                            minRows={1}
```
Comment (joaoviana) [inline packages/frontend/src/components/common/modal/DashboardCreateModal.tsx:155]: do we need to specify `minRows` here? 🤔

Comment (iMac7) [reply packages/frontend/src/components/common/modal/DashboardCreateModal.tsx:155]: Actually no, 1 row is the default

**Comment by owlas** [comment]
Closing while we wait for updates

**Pull request closed without merging**